### PR TITLE
added Auto-Anvil

### PR DESCRIPTION
**Pull request by MeerBiene**
Please tell me if the page doesnt meet your requirements, I edited this in the github web IDE so yeah :D

### CODE REVIEW

File: pages/Auto-Anvil [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+The auto anvil is one of the more complicated crafting recipes. You can find it under the category "Energy & Electricity" on the second page.
+
+# Tiers
+
+There are 2 tiers of this machine. Tier 1 has a Repair Factor of 10%, Tier2 20%.
+
+# Usage
+
+Place the item you want to repair together with "Duct Tape" in the machine and wait. (You need energy for it to work)
```
Comment (Phoenix-Starlight) [inline pages/Auto-Anvil:13]: The other method of usage would be to combine enchantment books with items. Also link duct tape item if it does exist.

File: pages/Auto-Anvil [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Auto-Anvil
```
Comment (Phoenix-Starlight) [inline pages/Auto-Anvil:1]: You do not need a header for the block/item name.

File: pages/Auto-Anvil [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+# Obtaining
+
+The auto anvil is one of the more complicated crafting recipes. You can find it under the category "Energy & Electricity" on the second page.
+
+# Tiers
+
+There are 2 tiers of this machine. Tier 1 has a Repair Factor of 10%, Tier2 20%.
```
Comment (Phoenix-Starlight) [inline pages/Auto-Anvil:9]: Minor typo, 'Tier2' should be 'Tier 2'

File: pages/Auto-Anvil [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Auto-Anvil
+
+# Obtaining
+
+The auto anvil is one of the more complicated crafting recipes. You can find it under the category "Energy & Electricity" on the second page.
```
Comment (Phoenix-Starlight) [inline pages/Auto-Anvil:5]: Maybe link to the Electric Machines category in the wiki.

**Review by Phoenix-Starlight — CHANGES_REQUESTED** [review]
Some changes and fixes suggested.

**Comment by MeerBiene** [comment]
Okay,time to edit this in a real editor. Should I also put a section about crafting or is that self explanatory?

**Comment by Phoenix-Starlight** [comment]
> Okay,time to edit this in a real editor. Should I also put a section about crafting or is that self explanatory?

They can find it in the in-game guide at /sf guide. So no.

**Comment by TheBusyBiscuit** [comment]
The file is also missing an ending (`.md`)

**Comment by svr333** [comment]
Just wanted to mention #93 also adds this page (and that it still hasnt gotten a second person to review it)

**Comment by TheBusyBiscuit** [comment]
Closed

**Pull request closed without merging**